compiler/qsc_eval/src/state.rs

```diff
@@ -483,7 +483,8 @@ fn write_latex_for_real_number(latex: &mut String, number: &RealNumber, render_o
 /// 1 is only rendered if ``render_one`` is true.
 fn write_latex_for_decimal_number(latex: &mut String, number: &DecimalNumber, render_one: bool) {
     if render_one || is_significant(number.value - 1.0) {
-        write!(latex, "{}", number.value).expect("Expected to write decimal value.");
+        write!(latex, "{}", (number.value * 10000.0).round() / 10000.0)
```

Comment:
Please refer to how the precision is set for formatting elsewhere in this file: 
https://github.com/microsoft/qsharp/blob/243ea46dd01bac3076f09a214ddb53782b09ce11/compiler/qsc_eval/src/state.rs#L27

You shouldn't have to round manually - Rust provides formatting specifiers for this purpose. `{:.4}` is the formatting specifier used to mean "4 digits after the decimal point". See Rust documentation to learn more: https://doc.rust-lang.org/std/fmt/#precision